addons/portal/i18n/portal.pot

```diff
@@ -266,21 +266,11 @@ msgstr ""
 msgid "Add extra content to display in the email"
 msgstr ""
 
-#. module: portal
-#: model_terms:ir.ui.view,arch_db:portal.portal_my_home
-msgid "Add, remove or modify your addresses"
```

Comment:
maybe the translation can be kept, because for most people (for all people that don't update)  the XMLwill still be in their templates